### ISSUE TITLE
TileMap get_local_mouse_pos() doesnt work properly on android

### ISSUE THREAD

**Issue by Solvero** [godotengine/godot#6441]
**Linux Mint, Windows - Godot version: 2.1**
**Xperia Neo V (Android 4.4.4) and Xperia T3**

**Issue description**:
I have simple scene with following structure:
-TileMap ([board.gd.zip](https://github.com/godotengine/godot/files/464702/board.gd.zip))
--selection

When I run it on the PC everything works as expected.
On the mobile phone I encounter strange behavior.
State switching seems to work fine since "selection" node blinks every screen touch, but
_put_ball() method is delayed/ random.

**Steps to reproduce:**
Create new project. Add TileMap then Sprite. Add attached script to the TileMap.
Run project on android via ADB.

**Comment by Solvero** [comment]
I have solved problem by myself. 
I don't know if its a bug, however it may be connected with issue "Touching the screen should trigger mouseover event too #1212".
I followed some advice to use "Android Device Monitor". In there I could read touch position live. I saw that get_local_mouse_pos() value is not updated every screen touch. So the put_ball() was setting one cell several times.
I have changed get_local_mouse_pos() method to event.pos and this gives me correct position every screen touch.
